Modules/Microsoft365DSC/Examples/Resources/AADAgreement/3-Remove.ps1

```diff
@@ -0,0 +1,23 @@
+<#
+This example removes an existing Azure AD Terms of Use Agreement.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [PSCredential]
+        $Credential
```

Comment:
Instead of defining $Credentials, we should be defining parameters $ApplicationId, $TenantId and $CertificateThumbprint. Then in the actual example below we should also be using those parameters instead of the $credential one.